### PR TITLE
Added loader and initial message

### PR DESCRIPTION
**Pull request by BhuvaneshPatil**
do mention any issue or problem this having I will fix it.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,6 +14,9 @@
     </div>
     <div id="subtitle">
     </div>
+    <div class="loader"></div>
+    <div class="message_portion"></div>
```
Comment (sbimochan) [inline index.html]: please make this message-portion. snakecase in css is not the right convention.

File: main.css [hunk 2]
```diff
@@ -60,4 +63,18 @@ body {
...
+  width: 50px;
+  height: 50px;
+  animation: spin 1s linear infinite;
+  margin: 20px auto;
+}
+
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
 }
```
Comment (sbimochan) [inline main.css]: add new line at end

File: index.html (hunk 1 above)
Comment (BhuvaneshPatil) [reply index.html]: > please make this message-portion. snakecase in css is not the right convention.

ok..working on it

File: script.js [hunk 3]
```diff
@@ -55,15 +58,23 @@ function startup() {
...
 				let resultHeight = video.videoHeight / (video.videoWidth / width);
-
+				
 				// Firefox currently has a bug where the height can't be read from
 				// the video, so we will make assumptions if this happens.
 
 				if (isNaN(resultHeight)) {
 					resultHeight = width / (4 / 3);
         }else {
+			const message_portion = document.querySelector('.message_portion');
```
Comment (sbimochan) [inline script.js]: duplicate declaration

**Review by sbimochan — CHANGES_REQUESTED** [review]

File: main.css [hunk 4]
```diff
@@ -10,13 +10,16 @@ body {
   background: #1D1F20;
 }
 
-#subtitle {
+#subtitle ,.message_portion{
```
Comment (sbimochan) [inline main.css]: looks like we can make both use same class i.e subtitle.

File: script.js (hunk 3 above)
Comment (BhuvaneshPatil) [reply script.js]: fixed that

File: main.css [hunk 5]
```diff
@@ -10,13 +10,16 @@ body {
   background: #1D1F20;
 }
 
-#subtitle {
+#subtitle ,.message-portion{
```
Comment (sbimochan) [inline main.css]: could you remove the id and make both same class?

**Comment by BhuvaneshPatil** [comment]
**1. removed double declaration
2. now no two different component for initial message and subtitle 
3. added line at the end of css file.**

File: index.html [hunk 6]
```diff
@@ -12,9 +12,12 @@
     <div class="gradient-border" id="camera">
       <video id="video">Video stream not available.</video>
     </div>
+    <div class="loader"></div>
     <div id="subtitle">
     </div>
+
```
Comment (sbimochan) [inline index.html]: remove unnecessary lines

File: main.css [hunk 7]
```diff
@@ -16,8 +16,10 @@ body {
   display: flex;
   align-items: center;
   justify-content: center;
+  padding-top: 20px;
 }
 
+
```
Comment (sbimochan) [inline main.css]: 2 extra lines. provide only single line

File: main.css [hunk 8]
```diff
@@ -60,4 +62,18 @@ body {
     100% {
         background-position: 0% 50%;
     }
-}
\ No newline at end of file
+}
```
Comment (sbimochan) [inline main.css:64]: add new line after it.

File: script.js [hunk 9]
```diff
@@ -55,17 +58,20 @@ function startup() {
         const width = 1024;
```
Comment (sbimochan) [inline script.js]: please also fix this wrong indentation

File: script.js [hunk 10]
```diff
@@ -29,11 +29,14 @@ const compliments = [
 	'You are making a difference.',
 	'You bring out the best in other people.',
 ];
-
+const message=document.querySelector("#subtitle");
```
Comment (sbimochan) [inline script.js]: add white space before and after "=" sign

Comment (sbimochan) [reply script.js]: can you name this variable subtitle instead?

File: index.html (hunk 6 above)
Comment (BhuvaneshPatil) [reply index.html]: removed

File: script.js (hunk 10 above)
Comment (BhuvaneshPatil) [reply script.js]: ok

**Comment by BhuvaneshPatil** [comment]
Done some changes have a look on it

File: index.html [hunk 11]
```diff
@@ -12,9 +12,10 @@
     <div class="gradient-border" id="camera">
       <video id="video">Video stream not available.</video>
     </div>
+    <div class="loader"></div>
     <div id="subtitle">
-    </div>
+    </div>
```
Comment (sbimochan) [inline index.html:17]:
```suggestion
    </div>
```

File: script.js [hunk 12]
```diff
@@ -45,32 +46,38 @@ function startup() {
 			video.play();
 		})
 		.catch(function (err) {
-			console.log('An error occurred: ' + err);
+			console.log("An error occurred: " + err);
 		});
 
 	video.addEventListener(
-		'canplay',
+		"canplay",
```
Comment (sbimochan) [inline script.js:53]: please donot change single quotes to double.

Comment (BhuvaneshPatil) [reply script.js:53]: It change due to extension of vs code (prettier).While formatting file

File: index.html (hunk 11 above)
Comment (BhuvaneshPatil) [reply index.html:17]: what to do here?

Comment (sbimochan) [reply index.html:17]: there was 2 unnecessary whitespace after div. remove it

**Pull request merged**